### PR TITLE
Improve documentation pmm docs

### PR DESCRIPTION
**Pull request by edithturn**
A minor error in the web page about PMM Architecture
- Fix the link for the Percona Platform
- Fix indentation for QAN API and QAN App

On other hand, I saw that  **azure_database_exporter** is not in the graphic but it is mentioned in the components of pmm-client. If this is correct that It should be in the image, I could add it! 💃🏼 
Screen Shot 2022-07-29 at 18 21 22

Another thing I was thinking is having this image as a **_mermaid_** (https://mermaid-js.github.io/mermaid/#/c4c) which is also Markdow, to make it maintainable, so if this image is as a code anyone could send PR to improve the image. It is just and idea :)

Any feedback is appreciated iI! Thank you!

### CODE REVIEW

**Comment by atymchuk** [comment]
> A minor error in the web page about PMM Architecture
> 
> * Fix the link for the Percona Platform
> * Fix indentation for QAN API and QAN App
> 
> On other hand, I saw that **azure_database_exporter** is not in the graphic but it is mentioned in the components of pmm-client. If this is correct that It should be in the image, I could add it! 💃🏼 Screen Shot 2022-07-29 at 18 21 22
> 
> Another thing I was thinking is having this image as a **_mermaid_** (https://mermaid-js.github.io/mermaid/#/c4c) which is also Markdow, to make it maintainable, so if this image is as a code anyone could send PR to improve the image. It is just and idea :)
> 
> Any feedback is appreciated iI! Thank you!

We have considered mermaid, but we decided not to adopt it simply because it is quite limited in features and therefore is not flexible enough. This is why we are opting to draw diagrams in miro.com. 

To your question about the missing exporter: that diagram is a reference architecture and we never pretended it could accommodate every component of our architecture - they are just too many :) So I'd leave it as it. By the way, we have a better digram and I'll propose to replace this one soon.

File: docs/details/architecture.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ PMM Client runs on every database host or node you want to monitor. The client c
 
 ## Percona Platform
 
-[Percona Platform](../using/platform/) provides value-added services for PMM.
+[Percona Platform](https://www.percona.com/software/percona-platform) provides value-added services for PMM.
```
[the remark this answers is not in the document]
Comment (rasika-chivate) [reply docs/details/architecture.md:21]: @atymchuk, if I understand correctly, this should point to our internal link. Let me just reaffirm with @catalina-adam.

[the remark this answers is not in the document]
Comment (rasika-chivate) [reply docs/details/architecture.md:21]: I think we should have an internal link. I will fix it.

[the remark this answers is not in the document]
Comment (atymchuk) [reply docs/details/architecture.md:21]: @rasika-chivate Is it done so we could close the ticket?

**Review by rasika-chivate — APPROVED** [review]

**Review by atymchuk — APPROVED** [review]

**Pull request merged**